netlify.toml

```diff
@@ -1,2 +1,2 @@
 [build]
-   ignore = "/bin/false"
\ No newline at end of file
+  ignore = "git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF . ../{docs,website}/"
```

Comment:
@TheJuanAndOnly99 , we tried quite a few combination of this - and ended up just by using /bin/false . If you figure out a working setup (that works for the draft pages too and for normal, etc), I am super interested.

Reply:
Hi @psmulovics not sure I understand what you mean by draft vs normal pages. The objective here is to only run Netlify on PRs that contain changes to the docs and website folders.